### PR TITLE
Use common get_original_issue implementation for all issue trackers (#504).

### PR DESCRIPTION
**Pull request by oliverchang**
Also, remove the merged_into setter, since it's not used by anything.

### CODE REVIEW

File: src/appengine/libs/issue_management/monorail/__init__.py [hunk 1]
```diff
@@ -234,10 +234,6 @@ def get_issue(self, issue_id):
 
     return Issue(monorail_issue)
 
-  def get_original_issue(self, issue_id):
-    """Retrieve the original issue object traversing the list of duplicates."""
-    return Issue(self._itm.get_original_issue(int(issue_id)))
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/monorail/__init__.py]: Probably i can remove some of these dead/duplicate functions in monorail part of code.
E.g. https://github.com/google/clusterfuzz/blob/master/src/appengine/libs/issue_management/monorail/issue_tracker_manager.py#L305. wdyt ? i can do this when i am back.

Comment (oliverchang) [reply src/appengine/libs/issue_management/monorail/__init__.py]: Yep, sgtm!

**Pull request merged**